amazon/ionbenchmark/ion_benchmark_cli.py

```diff
@@ -313,35 +352,61 @@ def ion_python_benchmark_cli(arguments):
     iterations = int(arguments['--iterations'])
     warmups = int(arguments['--warmups'])
     c_extension = str_to_bool(arguments['--c-extension']) if not pypy else False
-    binary = arguments['--format'] == Format.ION_BINARY.value
     iterator = str_to_bool(arguments['--iterator'])
 
+    # Note. For future multi-execution options, initialize them as below and added them into option_configuration.
+    # initialize options that might show up multiple times
+    api = [*set(arguments['--api'])] if arguments['--api'] else [API.DEFAULT.value]
+    format_option = [*set(arguments['--format'])] if arguments['--format'] else [Format.DEFAULT.value]
+    # option_configuration is used for tracking multi-execution options
+    option_configuration = [api, format_option]
+    option_configuration_combination = list(itertools.product(*option_configuration))
```

Comment:
For these options that might show up multiple times, do two things
1. Initialize them as [below](https://github.com/amazon-ion/ion-python/pull/235/files#diff-193f9a476a85a550a2aa80e3a9ad54272c2661544bc96455fd6439664e1b9df1R359),
2. Add into list `option_configuration`

That's it!